document-generation/generate-document.test.ts

```diff
@@ -219,12 +220,27 @@ describe("Successful generate-document handler", () => {
     expect(response.headers).toEqual(docHeaders.mrr);
   });
 
-  it("should return successful Eval Memo document response", async () => {
+  it("should return successful Eval Memo document response - w/ Exception to Fair Opportunity", async () => {
```

Comment:
This unit test was failing after I added it (but the other -- w/o Exception -- was passing).

Reply:
Believe I fixed it by correcting EM and EP schema models used by middy for validation when docgen lambda executes
https://github.com/dod-ccpo/atat-web-api/pull/1742/commits/83979dced2b31c729dc05e54e91c5a865aff5f8c